### DISCUSSION TITLE
1.1.0

### DISCUSSION THREAD

**Discussion by benhillis** [microsoft/WSL#9505] [General]
<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WSL/releases/tag/1.1.0'>1.1.0</a>.</em>

**Comment by klabacher** [comment]
Changelog?

**Reply by OneBlue** [reply]
Updated the release to include release notes. Thank you ! 

https://github.com/microsoft/WSL/releases/tag/1.1.0

**Comment by svetoslavenchev** [comment]
This version somehow interferes with Docker networking when I'm exposing container ports and binding them to 127.0.0.1:XXXX - I get connection refused when I try to connect. (Using Docker Desktop with WSL2 backend). The issue is not present in 1.0.3.
That's just a heads up, I'll try to open a proper issue with steps to reproduce later/tomorrow.

**Reply by StevenGFX** [reply]
Yup, I ran into this issue and ended up downgrading back to 1.0.3.

**Reply by damchey** [reply]
> Yup, I ran into this issue and ended up downgrading back to 1.0.3.

How do you downgrade? wsl --update --rollback does not work. Do I download an older package and install it?  This release really screwed up networking.

**Reply by Biswa96** [reply]
That command is probably for Linux kernel only. To downgrade, first uninstall the installed WSL, not the distros. Then download the older version from here https://github.com/microsoft/WSL/releases. Install the package with Add-AppxPackage command in Powershell.

**Reply by damchey** [reply]
> That command is probably for Linux kernel only. To downgrade, first uninstall the installed WSL, not the distros. Then download the older version from here https://github.com/microsoft/WSL/releases. Install the package with Add-AppxPackage command in Powershell.

Downloaded it from here and just ran it in Windows, it told me there was newer version install and if I wanted to proceed with reinstall. Did that it's working fine now with 1.0.3

By the way 1.1.0 is marked as "pre-release", I did not update using wsl --update --pre-release, it somehow updated through Microsoft Store. Wonder why it would do that.

**Reply by christostatitzikidis** [reply]
+1 for this.
Running VSCode devcontainers no longer maps ports correctly.
If you map a port in the devcontainer.json, the port will be unavailable in the container (Something is already using port xxx).
If you let it auto-map it will map to the next port. For example the docker port 3000, will be mapped to localhost 3001. 3001 to 3002 etc. etc.

Verified it's a WSL issue - Suddenly started happening on my desktop. It didn't occur on my laptop with same version of everything apart from WSL. As soon as I updated WSL on the laptop it started happening too.

![image](https://user-images.githubusercontent.com/396248/214341098-02a66ce1-708b-4b19-bda7-4fa1f707f52f.png)

**Reply by christostatitzikidis** [reply]
Confirming all working fine once rolled back to 1.0.3 as described above. Thanks for that

**Reply by christostatitzikidis** [reply]
BTW my update was automatic, through the store, (because I am on Beta Insider's Windows 11, I assume)

**Reply by astroboylrx** [reply]
> > Yup, I ran into this issue and ended up downgrading back to 1.0.3.
> 
> How do you downgrade? wsl --update --rollback does not work. Do I download an older package and install it? This release really screwed up networking.

The reply in this [issue](https://github.com/microsoft/WSL/issues/9508#issuecomment-1396866615) will help.

**Comment by JGCarroll** [comment]
Is the line

> Create dev/shm via bind mount (solves https://github.com/microsoft/WSL/issues/9384)

in the changelog pointing to the right bug? I feel like it should be bug #8996

**Reply by OneBlue** [reply]
Indeed ! Thank @MrCarroll  ! Updated the release notes

**Comment by bplasmeijer** [comment]
Hey @benhillis 

See, https://github.com/microsoft/WSL/issues/9508#issuecomment-1408166508 

Not a happy bug.

Thanks
Bart

**Reply by Cremesis** [reply]
Yeah, I'm experiencing that bug and it's not fun.
I suggest to wait for the next release and don't upgrade from 1.0.3 for now.

**Reply by Cremesis** [reply]
New release available I see, no more issues!

**Reply by christostatitzikidis** [reply]
Unfortunately still getting the same error here :(

![image](https://user-images.githubusercontent.com/396248/216291966-062a2f1d-c4a7-4db7-a7e4-389270140d57.png)
